docs/reference/contributing/target/itm.md

```diff
@@ -0,0 +1,34 @@
+### Instrumented Trace Macrocell
```

Comment:
Does this API have spaces in it, or does it follow the same formatting as the others ("InstrumentedTraceMacrocell")?

Reply:
I don't understand the question. Instrumented Trace Macrocell is the name of the module like Low Power Ticker.

Reply:
We have the ResetReason API, which is has no space and both Rs capitalized because we're referring to the class. Is it the same with this?

Reply:
Ah, no. This is a physical component defined in the architecture: http://infocenter.arm.com/help/index.jsp?topic=/com.arm.doc.ddi0314h/Chdegecg.html

Reply:
@marcuschangarm - Looks good, it would be great if you can add that ITM link in the documentation itself.  Its not a blocker though, so I'll approve.